index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Little Bird Web Development</title>
+  <link href="css/bootstrap.css" rel="stylesheet">
+  <link href="css/main.css" rel="stylesheet">
+  <link href='https://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet' type='text/css'>
+</head>
+<body>
+  <div class="navbar navbar-fixed-top well-lg">
+    <div class="container">
+      <div class="navbar-header">
+        <a href="#" class="navbar-brand">Little Bird Design</a>
+      </div>
+      <ul class="nav navbar-nav navbar-right">
+        <li><a href="">Portfolio</a></li>
+        <li><a href="">About</a></li>
+        <li><a href="">Contact</a></li>
+      </ul>
+    </div>
+  </div>
+  <div class="splash container-fluid">
+    <div class="logo">
+      <img src="images/logo4.png" alt="Little Bird Logo" class="img-circle img-responsive center-block">
+    </div>
+    <div class="container">
+      <h1 class="text-center nameplate">Jonathan Chickie</h1>
+      <h1 class="text-center"><s>----------</s><span class="glyphicon glyphicon-star"></span><s>----------</s></h1>
+      <h3 class="text-center subtitle">Web Developer - Graphic Artist - User Experience Designer</h3>
+    </div>
+  </div>
+
+  <div class="portfolio">
+    <div class="container">
+      <h1 class="text-center section-header">Portfolio</h1>
+      <h1 class="text-center"><s>---------</s><span class="glyphicon glyphicon-star"></span><s>---------</s></h1>
+    </div>
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-2 well well-lg col-md-offset-2"></div>
+        <div class="col-md-2 well well-lg col-md-offset-1"></div>
+        <div class="col-md-2 well well-lg col-md-offset-1"></div>
+      </div>
+      <div class="row">
+        <div class="col-md-2 well well-lg col-md-offset-2"></div>
+        <div class="col-md-2 well well-lg col-md-offset-1"></div>
+        <div class="col-md-2 well well-lg col-md-offset-1"></div>
+      </div>
+    </div>
+  </div>
```

Comment:
I condensed all of my empty `container` classes (now `container-fluid`) into their parent element (or an ancestor that worked) except for the two `container`s mentioned above!